enclave_manager/setup_singleton.py

```diff
@@ -82,7 +82,6 @@
     "avalon_enclave_manager/singleton/singleton_enclave.i",
     os.path.join(enclave_bridge_wrapper_path, 'swig_utils.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'work_order_wrap.cpp'),
-    os.path.join(enclave_bridge_wrapper_path, 'enclave_info.cpp'),
```

Comment:
Do we need to similar changes for seup_kme.py and setup_wpe.py ?

Reply:
yes.